### PR TITLE
Submission of ENMTools

### PR DESCRIPTION
**Pull request by danlwarren**
Added ENMTools.yaml.  Check commits on branch for details.

### CODE REVIEW

**Comment by sylvainschmitt** [comment]
occ_acquisition: no
occ_cleaning: trimdupes functions
data_integration: no
env_collinearity: raster.cor.matrix, raster.cor.plot. raster.pca
env_process: no
bias: modeling functions include bias estimates
study_region: background buffer functions define study region
backg_sample: included in all modeling functions and many hypothesis tests
data_partitioning: included in all modeling functions
mod_fit: included in all modeling functions
mod_tuning: enmtools.calibrate retunes models based on calibration on test data
mod_ensemble: not fully sure if this counts or not but the moses function compares merged and separate models for groups of species
mod_stack: no
mod_evaluate: included in all modeling functions plus enmtools.calibrate
mod_multispecies: moses functions as above, also hypothesis tests are sometimes used for this
mod_mechanistic: no
pred_general: all modeling functions
pred_extrapolation: all models can be predicted to new layers, plus threespace.plot is used for comparing training and extraoplation regions
pred_inspect: interactive plots, visualize.enm, marginal functions with all modeling methods
post_processing: lots of hypothesis tests
gui: no
metadata: no

**Comment by sylvainschmitt** [comment]
There are issue with your metadat compared to CRAN, pease fix:

>  -For package ENMTools, author is defined as 'Dan Warren and Russell Dunnage' but should be defined as 'Dan Warren, Russell Dinnage' according to CRAN.
-For package ENMTools, description is defined as 'Constructing niche models and analyzing patterns of niche evolution.  Acts as an interface for many popular modeling algorithms, and allows users to conduct Monte Carlo tests to address basic questions in evolutionary ecology and biogeography.' but should be defined as 'Constructing niche models and analyzing patterns of niche evolution. Acts as an interface for many popular modeling algorithms, and allows users to conduct Monte Carlo tests to address basic questions in evolutionary ecology and biogeography. Warren, D.L., R.E. Glor, and M. Turelli (2008) <doi:10.1111/j.1558-5646.2008.00482.x> Glor, R.E., and D.L. Warren (2011) <doi:10.1111/j.1558-5646.2010.01177.x> Warren, D.L

**Comment by sylvainschmitt** [comment]
I'm looking for reviewers: @EduardoArle , @gepinillab , @hannahlowens , @jamiemkass , @sgvignali, @codelab3 , @adamlilith; @MayaGueguen are you available? Please have a look at the [reviewing guidelines](https://github.com/sylvainschmitt/sdmverse/blob/main/REVIEWING.md)

**Comment by hannahlowens** [comment]
I can review.

File: inst/extdata/packages/ENMTools.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+name: ENMTools
+title: "Analysis of Niche Evolution using Niche and Distribution Models"
+version: 1.1.1
+author: "Dan Warren and Russell Dunnage"
```
Comment (hannahlowens) [inline inst/extdata/packages/ENMTools.yaml]:
```suggestion
author: "Dan Warren, Russell Dunnage"
```

**Review by hannahlowens — COMMENTED** [review]
Looks good, just one tiny suggestion.

File: inst/extdata/packages/ENMTools.yaml (hunk 1 above)
Comment (danlwarren) [reply inst/extdata/packages/ENMTools.yaml]: His last name is actually Dinnage, and I swear I corrected that before I submitted it.  Dammit.

**Comment by jamiemkass** [comment]
I can also review!

**Comment by sylvainschmitt** [comment]
> I can also review!

As you wish but @hannahlowens review can be enough. And I'm still searching reviewers for #19 .

File: inst/extdata/packages/ENMTools.yaml (hunk 1 above)
Comment (sylvainschmitt) [reply inst/extdata/packages/ENMTools.yaml]: This is problematic as we use CRAN metadata to validate the package metadata.

Comment (sylvainschmitt) [reply inst/extdata/packages/ENMTools.yaml]: In this case if you want to use metadata different from CRAN you need to change the cran field to no to stop automatic checking.

**Comment by jamiemkass** [comment]
Okay, I'll do #19 then.

**Comment by sylvainschmitt** [comment]
@danlwarren , thank you. Your contribution can be integrated without further edits. After merging your pull request, you will be added to the repository contributors. Please expect to be contacted for a review request in the future.

**Comment by danlwarren** [comment]
Thanks everyone!

**Pull request merged**